### ISSUE TITLE
Test failure with Python 3.5.2

### ISSUE THREAD

**Issue by felixonmars** [jaysonsantos/python-binary-memcached#50]
I am getting the following test error with Python 3.5.2, any hints?

```
=================================== FAILURES ===================================
______________________ TimeoutMemcachedTests.testTimeout _______________________

self = <test_simple_functions.TimeoutMemcachedTests testMethod=testTimeout>

    def testTimeout(self):
        self.client = bmemcached.Client(self.server, 'user', 'password',
                                        socket_timeout=0.00000000000001)
        self.client.set('timeout_key', 'test')
>       self.assertEqual(self.client.get('timeout_key'), None)
E       AssertionError: 'test' != None

test/test_simple_functions.py:225: AssertionError
===================== 1 failed, 104 passed in 1.72 seconds =====================
==> WARNING: Tests failed
```

**Comment by jaysonsantos** [comment]
It tests the socket timeout, you can either decrease the value of `socket_timeout` or mock the call just test it

**Comment by hathawsh** [comment]
This was fixed by #57.

**Comment by felixonmars** [comment]
Fixed in 0.25.0, closing. Thanks a lot!
